pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -8,217 +8,219 @@ package tests
+// cheap to generate for continous testing of the fixture generation logic.
+var SmallFixture = TpccFixture{
+	Name:                   "tpcc-5k",
+	ImportWarehouses:       5000,
+	WorkloadWarehouses:     1000,
+	IncrementalChainLength: 48,
+	RestoredSizeEstimate:   "350GiB",
+}
 
-type scheduledBackupSpecs struct {
-	backupSpecs
-	// ignoreExistingBackups if set to true, will allow a new backup chain
-	// to get written to an already existing backup collection.
-	ignoreExistingBackups    bool
-	incrementalBackupCrontab string
+// MediumFixture is a TPCC fixture sized so that it is a tight fit in 3 nodes
+// with the smallest supported node size of 4 VCPU per node.
+var MediumFixture = TpccFixture{
+	Name:                   "tpcc-30k",
+	ImportWarehouses:       30000,
+	WorkloadWarehouses:     5000,
+	IncrementalChainLength: 400,
+	RestoredSizeEstimate:   "2TiB",
 }
 
-func (sbs scheduledBackupSpecs) scheduledBackupCmd() string {
-	// This backup schedule will first run a full backup immediately and then the
-	// incremental backups at the given incrementalBackupCrontab cadence until the user cancels the
-	// backup schedules. To ensure that only one full backup chain gets created,
-	// begin the backup schedule at the beginning of the week, as a new full
-	// backup will get created on Sunday at Midnight ;)
-	options := ""
-	if !sbs.nonRevisionHistory {
-		options = "WITH revision_history"
-	}
-	backupCmd := fmt.Sprintf(`BACKUP INTO %s %s`, sbs.backupCollection(), options)
-	cmd := fmt.Sprintf(`CREATE SCHEDULE %s FOR %s RECURRING '%s' FULL BACKUP '@weekly' WITH SCHEDULE OPTIONS first_run = 'now'`,
-		scheduleLabel, backupCmd, sbs.incrementalBackupCrontab)
-	if sbs.ignoreExistingBackups {
-		cmd = cmd + ",ignore_existing_backups"
-	}
-	return cmd
+// LargeFixture is a TPCC fixture sized so that it is a tight fit in 3 nodes
+// with the maximum supported node density of 8 TiB storage per node. If the
+// node storage density increases, then the size of this fixture should be
+// increased.
+var LargeFixture = TpccFixture{
+	Name:                   "tpcc-300k",
+	ImportWarehouses:       300000,
+	WorkloadWarehouses:     10000,
+	IncrementalChainLength: 400,
+	RestoredSizeEstimate:   "20TiB",
 }
 
 type backupFixtureSpecs struct {
 	// hardware specifies the roachprod specs to create the scheduledBackupSpecs fixture on.
 	hardware hardwareSpecs
 
-	// scheduledBackupSpecs specifies the scheduled scheduledBackupSpecs fixture which will be created.
-	scheduledBackupSpecs scheduledBackupSpecs
-
-	// initWorkloadViaRestore, if specified, initializes the cluster via restore
-	// of an older fixture. The fields specified here will override any fields
-	// specified in the scheduledBackupSpecs field above.
-	initWorkloadViaRestore *restoreSpecs
+	fixture TpccFixture
 
 	timeout time.Duration
 	// A no-op, used only to set larger timeouts due roachtests limiting timeouts based on the suite
```

Comment:
I don't quite follow this comment. is it referring to the timeout field?

Reply:
I removed this comment. This comment was from the old version of the code. It previously made sense since fixture generation wasn't hooked up to either of the roachtest suites, so it only affected how long the timeout could be.